src/js/pages/system/OverviewDetailTab.js

```diff
@@ -43,13 +44,16 @@ class OverviewDetailTab extends mixin(StoreMixin) {
         name: 'metadata',
         events: ['dcosSuccess']
       }
-    ];
+    ]);
   }
 
   componentDidMount() {
     super.componentDidMount(...arguments);
     ConfigStore.fetchCCID();
     MarathonStore.fetchMarathonInstanceInfo();
+
+    Hooks.applyFilter('OverviewDetailTab:DidMountCallbacks', [])
```

Comment:
Allowing plugins to hook into a components `componentDidMount` callback feels kinda strange. Can't we use the `Mount` instead?